readme.md

```diff
@@ -0,0 +1,12 @@
+## Installation
```

Comment:
Lovely! Thanks for adding this in! Note: It still has some of the `confomo` entries in it.

Reply:
Oopsie! Updated!